Calculator/src/main/java/com/microsoft/device/display/samples/calculator/models/Enums.kt

```diff
@@ -0,0 +1,53 @@
+package com.microsoft.device.display.samples.calculator.models
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import com.microsoft.device.display.samples.calculator.R
+import com.microsoft.device.display.samples.calculator.ui.pages.calculatorModel
+
+// Equations that take multiple parameters
+enum class Equation {
+    ADD, SUB, MUL, DIV, POW, MOD, NONE,
+}
+
+// Actions and equations that only need a single parameter
+enum class Action {
+    DEL, CLR, EQL, DEG, INV, SIN, COS, TAN, LOG, LN, E, PI, SQRT, SQRD, SIGN, PRCT
+}
+
+@Composable
+fun equationStringEquivalent(eq: Equation): String {
+    return when (eq) {
+        Equation.ADD -> stringResource(id = R.string.add)
+        Equation.SUB -> stringResource(id = R.string.subtract)
+        Equation.MUL -> stringResource(id = R.string.multiply)
+        Equation.DIV -> stringResource(id = R.string.divide)
+        Equation.POW -> stringResource(id = R.string.power)
+        Equation.MOD -> stringResource(id = R.string.modulus)
+        else -> {
+            ""
+        }
```

Comment:
style comment but can simplify to one line as well
```suggestion
        else -> ""
```